### PR TITLE
material & home page update

### PR DESCRIPTION
**Pull request by rstorr**

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -1,8 +1,11 @@
 # Overview
 
-Codelingo is a Platform as a Service (PaaS) for software development teams to solve software development problems. It treats your software as data and automates your workflows, called Actions, with the rules and patterns you define, called Rules.
+Codelingo is a platform for software development teams to store, discuss and
+code their tacit knowledge into project specific rules.
```
Comment (Daanikus) [inline docs/index.md:4]:
```suggestion
CodeLingo is a platform for software development teams to store, discuss, and
codify their tacit knowledge into project specific rules.
```

Comment (Daanikus) [reply docs/index.md:4]: Thoughts on 'rules' vs 'Rules'? I feel like capitalising our own nomenclature helps to highlight their significance in sentences. Also, up to you on the Oxford comma ;)

File: docs/index.md [hunk 2]
```diff
@@ -1,8 +1,11 @@
...
 
-Our flagship Action is the [Review Action](https://www.codelingo.io/actions/codelingo/review), which checks a repository's pull requests conform to its project specific patterns. We have also built the [Rewrite Action](https://www.codelingo.io/actions/codelingo/rewrite) and the [Docs Action](https://www.codelingo.io/actions/codelingo/docs). 
+Our platform allows development teams to capture a discussion in a Pull Request,
+create a Rule around that discussion then use our DSL, CLQL to generate a
+project specific rule that is used to automatically review incoming code to your codebase.
```
Comment (Daanikus) [inline docs/index.md:8]:
```suggestion
Our platform allows development teams to capture a discussion in a Pull Request,
create a Rule around that discussion then use our DSL (CLQL: CodeLingo Query Language), to generate a
project specific rule that is used to automatically review incoming code to your codebase.
```

File: docs/index.md [hunk 3]
```diff
@@ -16,32 +19,16 @@ Test out writing a Rule and running a Action online with zero installs using the
 
 ## Dashboard
 
-The CodeLingo [Dashboard](https://www.codelingo.io/dashboard) lets you manage you automated workflows for each of your repositories. It is your hub for adding and removing Rules from your repositories and running Actions. Sign in with GitHub and follow the tutorial to run your first Action.
+The CodeLingo Dashboard
+[(codelingo.io/dashboard)](https://www.codelingo.io/dashboard) lets you manage
+your rules for each of your repositories. It is your teams hub for discussion and creating project specific rules.
```
Comment (Daanikus) [inline docs/index.md:24]:
```suggestion
The [CodeLingo Dashboard](https://dash.codelingo.io) lets you manage your rules for each of your repositories. It is your teams hub for discussing, creating, editing, and testing project specific rules.
```

File: docs/index.md [hunk 4]
```diff
@@ -16,32 +19,16 @@ Test out writing a Rule and running a Action online with zero installs using the
...
 
-## GitHub Review Action
+## Writing Rule Queries
 
-After [installing CodeLingo on GitHub](https://github.com/apps/codelingo), write the following codelingo.yaml to the root of your repository:
+Using CodeLingo Query Language (CLQL) you can write queries for your rules that
+will capture anti patterns in your codebase.
```
Comment (Daanikus) [inline docs/index.md:29]:
```suggestion
Using CodeLingo Query Language (CLQL) you can write queries for your rules that will capture anti-patterns in your codebase.
```

File: docs/index.md [hunk 5]
```diff
@@ -50,17 +37,15 @@ The CodeLingo platform has two key concepts: Rules and Actions.
 ## Rules
 
 A Rule is an encoded project specific best practice used to guide development. A Rule can be used for: coding styles, performance tuning, security audits, debugging, avoiding gotchas, reducing complexity and churn, and more.
+Rules can be created from the dashboard, you can either create a blank Rule or
+one from a Rule Template.
```
Comment (Daanikus) [inline docs/index.md:41]:
```suggestion
Rules can be created from the dashboard. You can either create a blank Rule or one from a Rule Template.
```

File: docs/index.md (hunk 1 above)
Comment (Daanikus) [reply docs/index.md:4]: Okay, after reading more, it seems you _are_ capitalising. There's just a few that you haven't. Can you please make those all consistent?

File: docs/index.md (hunk 3 above)
Comment (fwereade) [reply docs/index.md:24]: s/your teams hub/your team's hub/

**Pull request merged**